code generators/de.fhdo.lemma.model_processing.code_generation.container_base/src/main/java/de/fhdo/lemma/model_processing/code_generation/container_base/util/Util.xtend

```diff
@@ -0,0 +1,85 @@
+package de.fhdo.lemma.model_processing.code_generation.container_base.util
+
+import java.io.File
+import java.nio.file.Files
+import java.nio.file.Paths
+import java.util.regex.Pattern
+
+/**
+ * Utility class for the container base code generator.
+ *
+ * @author [Philip Wizenty](mailto:philip.wizenty@fh-dortmund.de)
+ */
+class Util {
```

Comment:
Please compare the functionalities within this class with what https://github.com/SeelabFhdo/lemma/blob/master/de.fhdo.lemma.utils/src/de/fhdo/lemma/utils/LemmaUtils.xtend offers already, and reuse the existing functionality from there or merge non-existing functionality there.